Taskfile.yml

```diff
@@ -63,21 +63,56 @@ tasks:
     desc: Run the unit tests.
     cmds:
       # One day I will understand how to use -coverpkg=./... :-(
-      - go test -count=1 -short -coverprofile=coverage.out ./...
+      # FIXME - go test -count=1 -short -coverprofile=coverage.out ./...
```

Comment:
This will be recovered in a subsequent PR (see commit that introduced this for details)